drag.js

```diff
@@ -93,6 +94,37 @@ Drag.prototype.setupEvents = function() {
 };
 
 /**
+ * Enables dragging.
+ *
+ * @public
+ */
+Drag.prototype.enable = function() {
+  this.enabled = true;
+};
+
+/**
+ * Disables dragging.
+ *
+ * @public
+ */
+Drag.prototype.disable = function() {
+  this.enabled = false;
+
+  if (this.point) {
+    this.translate(this.startPosition.x, this.startPosition.y);
+
+    this.point = null;
+    this.startPosition = null;
+
+    clearTimeout(this.timeout);
+    this.timeout = setTimeout(() => this.dragging = false);
+
+    removeEventListener(pointer.move, this.onPointerMove);
+    removeEventListener(pointer.up, this.onPointerEnd);
```

Comment:
Nit: Rather than duplicate the code, you could call onPointerEnd and not dispatch 'ended' if disabled?